### ISSUE TITLE
consider adding docstrings

### ISSUE THREAD

**Issue by yashaka** [yashaka/selene#59]
I am still unsure whether we need them everywhere. Some methods are already obvious in usage...

**Comment by aleksandr-kotlyar** [comment]
for example
```
browser.quit
browser.close
```

**Comment by yashaka** [comment]
Yeah, for some commands it's good to have docstrings:)
like for the ones you mentioned.

**Comment by MatuMikey** [comment]
It's pretty good practice to have docstrings for methods - helps a lot with IntelliSense in most IDE's. It's always nice to have, it looks good too. Some of the most popular libraries out there have comments for the most obvious methods. Let us not assume everyone is intelligent enough to figure it out :).

If you would like help with docstrings for methods for documenting everything, I will be more than glad to help @yashaka !

**Comment by yashaka** [comment]
@MatuMikey 

I am not sure "most popular libraries" should be used as authority to follow:) I believe we should filter "popularity" by common sense :)

Probably the majority of Selene methods would benefit from having docstrings... If the minority of methods don't need docstrings, then because of "minority" – we still can add docstrings for consistency... But if "half of selene methods" (i.e. not minority) have the context of "nothing useful can be added as docstring", then I would not bloat code with something useless just for consistency. Nevertheless, we have to start adding usefull docstrings first, to see - how much "obvious methods" will remain. Then we can decide what to do with them:) Maybe it will happen that something useful can be added to docstrings of all selene methods:)

Right now I working on #406, that is pretty big chunk of work... Probably we should start working on docstrings after #406 is done...

**Comment by yashaka** [comment]
connected to #456
